.github/PULL_REQUEST_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,33 @@
+## Summary
+
+Brief description of the documentation update and its purpose.
+
+## Areas Affected
+
+List the sections or pages of the documentation that are updated.
+
+## Details
+
+Provide detailed information about the changes made to the documentation.
+
+## Motivation
+
+Explain why these documentation changes are necessary.
```

Comment:
Seems a bit superfluous to have 4 entries? I'm ok relying on code changes for some of the details. I think a single summary should suffice?